pymeasure/instruments/signalrecovery/dsp7265.py

```diff
@@ -166,13 +158,41 @@ class DSP7265(Instrument):
         values=REFERENCES,
         map_values=True
     )
-    sensitivity = Instrument.control(
-        "SEN", "SEN %d",
+
+    @property
+    def sensitivity(self):
         """ A floating point property that controls the sensitivity
-        range in Volts, which can take discrete values from 2 nV to
-        1 V. This property can be set. """,
-        validator=truncated_discrete_set,
-        values=SENSITIVITIES,
+        range in Volts (for voltage mode) or Amps (for current modes).
+        When in Volts it takes discrete values from 2 nV to 1 V. When in
+        Amps it takes discrete values from 2 fA to 1 µA (for normal current
+        mode) or up to 10 nA (for low noise current mode).
+        This property can be set; setting requires a call to `imode`. """
```

Comment:
"setting requires a call to `imode`." maybe delete that, since it is not something relevant for the user, and could cause confusion?

Reply:
Yes, I'll remove it.